### ISSUE TITLE
Ressources in mountains are reseted for new mine

### ISSUE THREAD

**Issue by Embeddix** [unknown-horizons/unknown-horizons#2340]
A mountains has a limited amount of iron, let's say it's 400. After all of this has been digged out of the mountain there should be no ressources left. Currently yu just have to build a new mine and there again 400 iron.

Steps to reproduce:
1.) Build an iron mine on a mountain
2.) Dig out all of the iron (0 Iron left)
3.) Pull down the mine
4.) Build a new mine (again 400 iron there).

At step 4 there should be still 0 iron.

**Comment by LinuxDonald** [comment]
I will check that in the lastet git version. But i think that bug will not get fixed up in Version 2015.1 when its still there.

**Comment by AndyMender** [comment]
I will check whether the issue is reproducible still and generate a test savegame that later be shared.

**Comment by AndyMender** [comment]
@LinuxDonald, @ChrisOelmueller, @squiddy, what do you think about the current resource deposit mechanics? Should the resources be respawnable and endless or should each deposit contain a fixed amount of a resource type, which is gone after depletion?

I removed the "bad usability" tag, because it's a game design thing and not a side-effect.

**Comment by LinuxDonald** [comment]
I remeber that on Anno 1602 an Mountain had fixed Resources? But what is when an player play free to play mode/endless mode?
I think it should be an game option before you start an game what you want. Fixed ressources or endless. And for multiplayer its an setting that can makes alot different play sytle.

**Comment by AndyMender** [comment]
@LinuxDonald, should this option be added to one of the game menu widgets and by default be set to "ON"? I could later implement a True/False trigger where the resource deposit loading bit is.

**Comment by LinuxDonald** [comment]
Yes it should be an game option. Selectable before the gamer start the game. (Free-to-play)
